.drone.yml

```diff
@@ -42,23 +42,33 @@ pipeline:
       - cd /drone/server; php ./occ a:l
       - cd /drone/server; php ./occ a:e files_primary_s3
       - cd /drone/server; php ./occ a:l
-      - cp /drone/server/apps/files_primary_s3/tests/drone/scality.config.php /drone/server/config
+    when:
+      local: false
 
-  test-phpunit:
+  test-scality:
     image: owncloudci/php:7.1
     pull: true
     environment:
-      - DATABASE_HOST=mariadb
-      - DATABASE_NAME=owncloud
-      - DATABASE_USER=owncloud
-      - DATABASE_PASSWORD=owncloud
-      - COMPOSER_CACHE_DIR=/drone/src/composer
+      - COMPOSER_CACHE_DIR=/drone/composer
```

Comment:
This should be `/drone/src/composer` or however yu want to name the dir, but it must be part of the workspace path.

Reply:
ok - hmmmm ..... but I move src around - I assume this will break - lets work on this once #2 is working